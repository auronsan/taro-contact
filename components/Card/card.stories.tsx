import React from 'react';
import { Card } from './index';

export default {
  title: 'components/Card',
  component: Card,
};

export const WithContent = () => <Card>Content</Card>;
