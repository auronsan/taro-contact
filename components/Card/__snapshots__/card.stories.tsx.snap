// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/Card WithContent smoke-test 1`] = `
<main class="roboto-normal">
  <div class="card_card__sba1Z">
    <div>
      Content
    </div>
  </div>
  <div class="stack_root__KVCbK toast_toast-container__8fZtQ stack_gap-xs__xTnny"
       id="toast-container"
       data-testid="toast-container"
       style="padding: 0px;"
  >
  </div>
</main>
`;
